server-base/src/main/java/org/apache/kylin/rest/service/QueryService.java

```diff
@@ -297,6 +297,12 @@ public void logQuery(final String queryId, final SQLRequest request, final SQLRe
             }
         }
 
+        if (realizationNames.isEmpty()) {
+            if (!Strings.isNullOrEmpty(response.getCube())) {
+                realizationNames.addAll(Lists.newArrayList(response.getCube().split(",")));
```

Comment:
Use CollectionUtils.addAll to avoid newArrayList

Reply:
agree